.travis.yml

```diff
@@ -7,17 +7,13 @@ node_js:
   - 10
   - 8
 env:
-  - FRESH_DEPS=false
   - FRESH_DEPS=true
 matrix:
-  exclude:
+  include:
+    # Only verify package-lock.json on node.js 11 for linux
```

Comment:
This isn't about verifying the package lock, or at least not how I would understand it. The idea is that the package lock gives us a predictable CI run, but we do at least one run with the newest dependencies. Probably more useful for a bigger project like AVA than this module though.

Reply:
The problem is that it does verify the package-lock.json and the file is different between node.js 11 and all previous versions.

Reply:
Ah yes of course. Let's just delete the package lock and remove the CI checks. Best disable it too (like https://github.com/sindresorhus/p-defer/blob/master/.npmrc).

Reply:
Done